docker-images/postgresql/initdb-data/50-genomicannotations.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
This part needs to be modified.

The postgresql image should create only the database and the role needed to connect to it, i.e. this part:
```
CREATE DATABASE "$element";
ALTER DATABASE "$element" OWNER TO genomicannotations;
```
(as it was before)

The content of the database should be created from sql script in the medco-connector, with a structure similar to the i2b2 docker image (sql folder containing the sql scripts, and in the docker-entrypoint.sh add the part that checks if the db exists and if not initializes it).

This structure is needed so that we are flexible in terms of deployment (1 DB for several medco on one machine, 1 DB per medco node, etc.), and that it is kept consistent with the rest of the other images.